Gemfile

```diff
@@ -1,6 +1,6 @@
 source 'https://rubygems.org'
 
-ruby '>= 2.4.4'
+ruby '>= 2.4.0'
```

Comment:
why the downgrade here?

Reply:
In `.travis.yml` I've set travis to use Ruby 2.4.1 because this is pre-installed on Travis, and should be faster.

The downgrade here will accommodate that, but still allow us to run 2.4.4